src/legacy/core_plugins/console/public/src/directives/_history.scss

```
@@ -1,3 +1,7 @@
+sense-history {
+  padding: 8px;
```

When adding SASS, please use EUI sass variables instead of hard-coded px values. We have a [guides page on SASS](http://localhost:8030/#/guidelines/sass) to help you out.

For this one it would be:
```suggestion
  padding: $euiSizeS;
```